.env_SAMPLE

```diff
@@ -51,11 +51,26 @@ export ALLOW_ADMIN_URL=True
 export DATABASE_URL=sqlite:///db.sqlite3
 #export DATABASE_URL=postgres://cfpb@localhost/cfgov
 
-# The use of default_storage_engine here assumes use of MySQL 5.7. For
-# MySQL 5.5 or earlier, use storage_engine instead.
+#DEPRECATED:
 #export DATABASE_URL=mysql://root@localhost/v1?init_command='SET default_storage_engine=MYISAM'
 
-# Required for existing MySQL-specific database scripts.
+#########################################################################
+# Postgres env vars to make tools like psql easier to work with
+#
+# See https://www.postgresql.org/docs/current/static/libpq-envars.html
+#########################################################################
+
+#export PGHOST=localhost
+#export PGDATABASE=cfgov
+#export PGUSER=cfpb
+#export PGPASSWORD=cfpb
```

Comment:
How would you feel about removing this line and encouraging use of a password-less `cfpb` user locally? See above on line 52 where we don't provide a password.

I realize that the Docker setup does use a password.

Reply:
@chosak Good call, I think – as you point out – I did that just to make it match what's in `docker-compose.yml`, but let's go for password-less outside of Docker as the default, will fix 👍